addons/l10n_dk/data/account_tax_template_data.xml

```diff
@@ -553,7 +604,8 @@
         <field name="sequence">710</field>
         <field name="name">Olie- og flaskegasafgift</field>
         <field name="amount">100</field>
-        <field name="amount_type">percent</field>
+        <field name="amount_type">division</field>
+        <field name="price_include" eval="True"/>
```

Comment:
a division with price include and a 100%?  You seem to exaggerate a bit.  Would not it just work with percentage as well?

Reply:
![image](https://user-images.githubusercontent.com/52834046/152215485-233f98a1-7d18-42ae-a2c9-e4ebe57365d2.png)
![image](https://user-images.githubusercontent.com/52834046/152215638-68f28552-f756-4e7b-b530-2ef50a299eba.png)

Reply:
If I keep price_include, the difference between division and percentage is having the tax correct or wrong.

If we want to use the classic percentage, then we have to let it the way it is with 3 repartition lines and with 2 that are in opposite way.

I know using division is particularly unusual and price_include can be guilty of a lot of rounding approximation but the use-case itself is very unusual and making it that way helps to make it clearer.

First, on the invoice, the untaxed and taxed amount that are displayed are 5000 and 2000 instead of 7000 and 0 with the current implementation.
In the amls, it makes less noise by avoiding creating an aml on the credit side of the elafgift account.